qgis/tests/test_io.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from qgis.testing import unittest
+
+
+class TestIO(unittest.TestCase):
+    def test_passes(self):
+        self.assertTrue(True)
+
+    # TODO Open actual geopackage
+    def test_open_geopackage(self):
+        pass
```

Comment:
I think we should wait with merging this PR until it actually tests something :)

Reply:
This actually does test that the plugin is installable and can load in QGIS.

Reply:
I see.

Does that also work if you remove the dummy tests?

Reply:
And, besides being timeboxed, this does provide the quite generic boilerplate that our other X-qgis packages can use.